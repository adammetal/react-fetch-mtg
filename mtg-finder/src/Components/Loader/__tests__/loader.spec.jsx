import { render } from '@testing-library/react';
import Loader from '../index';

test('smoke test for loader', () => {
  const { container } = render(<Loader />);
  expect(container).toMatchSnapshot();
})
