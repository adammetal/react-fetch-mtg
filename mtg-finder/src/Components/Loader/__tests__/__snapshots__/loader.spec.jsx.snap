// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test for loader 1`] = `
<div>
  <div
    class="loader"
  />
</div>
`;
